src/main/java/tutorease/address/model/person/Role.java

```diff
@@ -7,10 +7,12 @@
  * Guarantees: immutable; its value is either "Guardian" or "Student"
  */
 public class Role {
+    public static final String GUARDIAN_LOWERCASE = "guardian";
+    public static final String STUDENT_LOWERCASE = "student";
     public static final String GUARDIAN = "Guardian";
     public static final String STUDENT = "Student";
     public static final String MESSAGE_CONSTRAINTS = "Roles can take 'Guardian' or 'Student', and it should not be "
-            + "blank";
+            + "blank." + "Also, roles are case-insensitive.";
```

Comment:
I think that this is too much information. I think it will be better to put it in our DG instead

Reply:
Sure!